content/en/firewall.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Firewall
+status: Feedback Appreciated
```

Comment:
(Reminder for all) When this term gets okay to be displayed in CNCF Glossary website,
please change the `status` to `Completed`. 😊

```suggestion
status: Completed
```